AppServices/Services/CustomerAppService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using AppServices.Interfaces;
+using DomainModels;
+using DomainServices.Interfaces;
+using System;
+using System.Collections.Generic;
+
+namespace AppServices.Services
+{
+    public class CustomerAppService : ICustomerAppService
+    {
+        private readonly ICustomerService _service;
+        public CustomerAppService(ICustomerService service)
+        {
+            _service = service ?? throw new ArgumentNullException(nameof(service));
+        }
+
+        public virtual void Create(Customer customer)
+        {
+            _service.Create(customer);
+        }
+
+        public virtual void Delete(long id)
+        {
```

Comment:
[Comentário Acima](https://github.com/LuanFortunatoW/EverestBackendDotnet/pull/2/files#r1027900637)
```suggestion
        public void Delete(long id)
        {
```